content/en/continuous_integration/quality_gates/_index.md

```diff
@@ -0,0 +1,175 @@
+The default site is US1 (datadoghq.com).
+
+{{< code-block lang="shell" >}}
+DD_BETA_COMMANDS_ENABLED=true DATADOG_API_KEY=<API_KEY> DATADOG_APP_KEY=<APP_KEY> datadog-ci gate evaluate
+{{< /code-block >}}
+
+The behavior of the command can be modified using the following flags:
+- **--fail-on-empty**: when this flag is specified, the command fails if no matching rules were found in Datadog
+based on the current command scope. By default, the command succeeds.
+- **--fail-if-unavailable**: when this flag is specified, the command fails if one or more rules could not be evaluated.
+By default, the command succeeds.
+- **--no-wait**: by default, the command waits a certain amount of time for the events (tests, static analysis violations) to arrive to Datadog.
+This step is important as it makes sure that the events are queryable in Datadog before the rules are executed,
+avoiding incorrect evaluation. If, in your pipeline, the job containing the `datadog-ci gate evaluate` command is
+called several minutes after the related events are sent to Datadog, you could skip this waiting time by specifying the `--no-wait` flag.
+Note that, if used incorrectly, this flag might result in inaccurate rule evaluations.
+
+[Custom scopes](#custom-scope) can be added by using the following option one or more times: **--scope**:
+
+{{< code-block lang="shell" >}}
+datadog-ci gate evaluate --scope team:backend --scope team:frontend
+{{< /code-block >}}
+
+The command logs can be examined to understand what was the overall gate evaluation status, along with information
+about all the rules that were evaluated.
+
+{{< img src="ci/datadog_ci_gate_evaluate_logs.png" alt="Datadog-ci gate evaluate logs" style="width:90%;">}}
+
+## Rule scope
+
+When creating a rule, you can define its scope, which states when it should be evaluated.
+The rule scope is then checked based on the context passed by the `datadog-ci gate evaluate` command to understand whether the rule should be evaluated or not.
+
+For each scope name (for example, "branch"), you can either select included or excluded values.
+When included values are selected, the rule is evaluated if one or more included values are part of the command context.
+When excluded values are selected, the rule is evaluated if none of the excluded values are part of the command context.
+
+For example, to create a rule that is evaluated in all branches but `main` of the `example-repository` repository, you can define the following scope:
+
+{{< img src="ci/scope_not_main_example_repository.png" alt="Rule scope for example-repository and not main branch" style="width:90%;">}}
+
+If a rule does not contain a scope name, it is evaluated for all values of that scope name.
+For example, if a rule does not contain the `repository` scope, it is evaluated for all repositories. If the rule scope is
+set to `always evaluate`, the rule is evaluated on all repositories and branches.
```

Comment:
should we move it somewhere in the beginning of the section, because it's what we show by default in the UI?

Reply:
I have moved this to the beginning. I'll check later if it's better there